### DISCUSSION TITLE
webdav问题

### DISCUSSION THREAD

**Discussion by creeper2020** [AlistGo/alist#883] [Q&A]
A机安装alist添加本地目录后经测试使用A机的webdav可以访问，B机安装alist通过alist挂载类型挂载该目录后无法通过B机的webdav访问，B机通过网页端可以访问挂载的A机的内容，A,B机安装的alist均为最新版。B机的日志如下DEBU[2022-04-01 12:04:42]/source/model/account.go:135 github.com/Xhofe/alist/model.GetBalancedAccount() accounts: []                                 
[GIN] 2022/04/01 - 12:04:42 | 200 |     422.466µs |     111.27.5.77 | POST     "/api/public/path"
DEBU[2022-04-01 12:04:43]/source/model/account.go:135 github.com/Xhofe/alist/model.GetBalancedAccount() accounts: [{ID:4 Name:hdd Index:0 Type:Alist Username: Password: RefreshToken: AccessToken:1c575b4de9832b11********** RootFolder:/ Status:work CronId:0 DriveId: Limit:0 OrderBy: OrderDirection: UpdatedAt:2022-04-01 12:04:38.885997386 +0000 UTC Search:false ClientId: ClientSecret: Zone: RedirectUri: SiteUrl:http://hdd.mymoe.in SiteId: InternalType: WebdavProxy:true Proxy:true WebdavDirect:false DownProxyUrl: APIProxyUrl: Bucket: Endpoint: Region: AccessKey: AccessSecret: CustomHost: ExtractFolder: Bool1:false}] 
DEBU[2022-04-01 12:04:43]/source/server/common/files.go:21 github.com/Xhofe/alist/server/common.Path() use account:  hdd                            
[GIN] 2022/04/01 - 12:04:44 | 200 |  563.870795ms |     111.27.5.77 | POST     "/api/public/path"
DEBU[2022-04-01 12:04:46]/source/model/account.go:135 github.com/Xhofe/alist/model.GetBalancedAccount() accounts: []                                 
[GIN] 2022/04/01 - 12:04:46 | 200 |     384.047µs |     111.27.5.77 | POST     "/api/public/path"
DEBU[2022-04-01 12:05:50]/source/model/account.go:135 github.com/Xhofe/alist/model.GetBalancedAccount() accounts: [{ID:4 Name:hdd Index:0 Type:Alist Username: Password: RefreshToken: AccessToken:1c575b4de9832b1********* RootFolder:/ Status:work CronId:0 DriveId: Limit:0 OrderBy: OrderDirection: UpdatedAt:2022-04-01 12:04:38.885997386 +0000 UTC Search:false ClientId: ClientSecret: Zone: RedirectUri: SiteUrl:http://hdd.mymoe.in SiteId: InternalType: WebdavProxy:true Proxy:true WebdavDirect:false DownProxyUrl: APIProxyUrl: Bucket: Endpoint: Region: AccessKey: AccessSecret: CustomHost: ExtractFolder: Bool1:false}] 
DEBU[2022-04-01 12:05:50]/source/server/common/files.go:21 github.com/Xhofe/alist/server/common.Path() use account:  hdd                            
[GIN] 2022/04/01 - 12:05:50 | 200 |     570.609µs |     111.27.5.77 | POST     "/api/public/path"
DEBU[2022-04-01 12:06:23]/source/model/account.go:135 github.com/Xhofe/alist/model.GetBalancedAccount() accounts: []                                 
DEBU[2022-04-01 12:06:23]/source/server/webdav/file.go:75 github.com/Xhofe/alist/server/webdav.(*FileSystem).Files() admin                                        
[GIN] 2022/04/01 - 12:06:23 | 207 |     346.657µs |     111.27.5.77 | PROPFIND  "/dav/"
DEBU[2022-04-01 12:06:24]/source/model/account.go:135 github.com/Xhofe/alist/model.GetBalancedAccount() accounts: [{ID:4 Name:hdd Index:0 Type:Alist Username: Password: RefreshToken: AccessToken:1c575b4de9832b11e5************ RootFolder:/ Status:work CronId:0 DriveId: Limit:0 OrderBy: OrderDirection: UpdatedAt:2022-04-01 12:04:38.885997386 +0000 UTC Search:false ClientId: ClientSecret: Zone: RedirectUri: SiteUrl:http://hdd.mymoe.in SiteId: InternalType: WebdavProxy:true Proxy:true WebdavDirect:false DownProxyUrl: APIProxyUrl: Bucket: Endpoint: Region: AccessKey: AccessSecret: CustomHost: ExtractFolder: Bool1:false}] 
DEBU[2022-04-01 12:06:24]/source/server/webdav/webdav.go:49 github.com/Xhofe/alist/server/webdav.isPathExist() /hdd path not found                          
[GIN] 2022/04/01 - 12:06:24 | 404 |     151.392µs |     111.27.5.77 | PROPFIND  "/dav/hdd/"

**Comment by xhofe** [comment]
...直接连接不好吗 为什么非要套娃

**Reply by creeper2020** [reply]
直连速度不行啊，一个大盘加一个线路还行的机子体验更好点，之前版本都没问题

**Reply by xhofe** [reply]
已修复，但是暂时不会发新版本。你可以：
- 在[Actions](https://github.com/Xhofe/alist/actions/runs/2077324921)中下载开发版本
- 使用[alist-proxy](https://github.com/alist-org/alist-proxy)而不是套娃（推荐）

**Reply by creeper2020** [reply]
感谢
